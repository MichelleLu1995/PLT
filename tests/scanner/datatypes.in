String
int
bool
float
void
