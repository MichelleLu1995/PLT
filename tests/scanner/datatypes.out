STRING
INT
BOOL
FLOAT
VOID
